src/server/game/Entities/Creature/Creature.cpp

```diff
@@ -1464,6 +1455,15 @@ bool Creature::CreateFromProto(ObjectGuid::LowType guidlow, uint32 entry, Creatu
     if (!UpdateEntry(entry, data))
         return false;
 
+    UpdatePositionData();
+    SetZoneScript();
+    if (GetZoneScript() && data)
+    {
+        entry = GetZoneScript()->GetCreatureEntry(guidlow, data);
```

Comment:
great, you completely broke GetCreatureEntry hook.

Reply:
Will still trigger an assert then, doing it in UpdateEntry doesnt feel right to me as we are not really updating a position. Reverting it and leaving the default collisionheight deal with it sounds like a better option to me